pykickstart/commands/raid.py

```diff
@@ -414,16 +499,43 @@ def __init__(self, writePriority=131, *args, **kwargs):
 
         self.levelMap.update({"RAID10": "RAID10", "10": "RAID10"})
 
+    def _getParser(self):
+        op = FC5_Raid._getParser(self)
+        for action in op._actions:
+            if "--level" in action.option_strings:
+                action.help += dedent("""
+
+                .. versionchanged:: %s
+
+                The "RAID10" level was added.""" % \
+                versionToLongString(F7))
+                break
+        return op
+
 class F9_Raid(F7_Raid):
     removedKeywords = F7_Raid.removedKeywords
     removedAttrs = F7_Raid.removedAttrs
 
     def _getParser(self):
         op = F7_Raid._getParser(self)
-        op.add_argument("--bytes-per-inode", deprecated=True)
```

Comment:
Yes, that's why these tests were failing earlier for me. Thanks for catching it.